### PR TITLE
[BULK] DocuTune - Updates to Azure AD rebranding guidance and DocuTune configuration (part 2)

### PR DESCRIPTION
**Pull request by alexbuckgit**
Applying Azure AD rebranding changes as announced on July 11, 2023 and outlined in https://aka.ms/AzureADNewName. Changes are part of the Microsoft-wide rebranding effort.

CorrelationId: 2520d58e-25ff-469e-af14-2834d2a1505e
PointOfContact: @CelesteDG

#docutune

### CODE REVIEW

**Comment by alexbuckgit** [comment]
#docutune-draft

File: src/Compute/Compute/help/Get-AzVMADDomainExtension.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ Get-AzVMADDomainExtension [-ResourceGroupName] <String> [-VMName] <String> [-Nam
 ```
 
 ## DESCRIPTION
-The **Get-AzVMADDomainExtension** cmdlet gets information about the specified Azure Active Directory (AD) domain extension.
+The **Get-AzVMADDomainExtension** cmdlet gets information about the specified Microsoft Entra ID \(AD) domain extension.
```
Comment (mikefrobbins) [inline src/Compute/Compute/help/Get-AzVMADDomainExtension.md:22]: The added backslash `\` is unnecessary and `(AD)` should be removed.

File: src/Compute/Compute/help/Set-AzVMADDomainExtension.md [hunk 2]
```diff
@@ -21,7 +21,7 @@ Set-AzVMADDomainExtension -DomainName <String> [-OUPath <String>] [-JoinOption <
 ```
 
 ## DESCRIPTION
-The **Set-AzVMADDomainExtension** cmdlet adds an Azure Active Directory (AD) domain virtual machine extension to a virtual machine.
+The **Set-AzVMADDomainExtension** cmdlet adds a Microsoft Entra ID \(AD) domain virtual machine extension to a virtual machine.
```
Comment (mikefrobbins) [inline src/Compute/Compute/help/Set-AzVMADDomainExtension.md:24]: Same issue. The added backslash `\` is unnecessary and `(AD)` should be removed.

**Review by mikefrobbins — CHANGES_REQUESTED** [review]
Please see my comments.

**Comment by CelesteDG** [comment]
@alexbuckgit This PR needs your attention

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 3]
```diff
@@ -110,7 +110,9 @@ $params | Set-AzVMDiskEncryptionExtension
 
 This example sends parameters using pipelined input to enable encryption on a VM, without specifying AD credentials.
 
-### Example 3: Enable encryption using Azure AD Client ID and Client Secret
+<a name='example-3-enable-encryption-using-azure-ad-client-id-and-client-secret'></a>
+
```
Comment (BethanyZhou) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md:114]:
```suggestion
```

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 4]
```diff
@@ -124,9 +126,11 @@ $VolumeType = "All"
 Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -AadClientID $AADClientID -AadClientSecret $AADClientSecret -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId -VolumeType $VolumeType
 ```
 
-This example uses Azure AD client ID and client secret to enable encryption on a VM.
+This example uses Microsoft Entra client ID and client secret to enable encryption on a VM.
+
+<a name='example-4-enable-encryption-using-azure-ad-client-id-and-client-certification-thumbprint'></a>
```
Comment (BethanyZhou) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md:131]:
```suggestion
```

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 5]
```diff
@@ -177,9 +181,11 @@ Update-AzVM -VM $VM -ResourceGroupName $RGName
 Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -AadClientID $AADClientID -AadClientCertThumbprint $AADClientCertThumbprint -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId -VolumeType $VolumeType
 ```
 
-This example uses Azure AD client ID and client certification thumbprints to enable encryption on a VM.
+This example uses Microsoft Entra client ID and client certification thumbprints to enable encryption on a VM.
+
+<a name='example-5-enable-encryption-using-azure-ad-client-id-client-secret-and-wrap-disk-encryption-key-by-using-key-encryption-key'></a>
```
Comment (BethanyZhou) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md:186]:
```suggestion
```

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 6]
```diff
@@ -124,9 +124,11 @@ $VolumeType = "All"
 Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -AadClientID $AADClientID -AadClientSecret $AADClientSecret -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId -VolumeType $VolumeType
 ```
 
-This example uses Azure AD client ID and client secret to enable encryption on a VM.
+This example uses Microsoft Entra client ID and client secret to enable encryption on a VM.
 
-### Example 4: Enable encryption using Azure AD client ID and client certification thumbprint
+<a name='example-4-enable-encryption-using-azure-ad-client-id-and-client-certification-thumbprint'></a>
+
```
Comment (BethanyZhou) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md:130]:
```suggestion
```

File: src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md [hunk 7]
```diff
@@ -200,9 +200,11 @@ $KeyEncryptionKeyUrl = $KEK.Key.kid
 Set-AzVMDiskEncryptionExtension -ResourceGroupName $RGName -VMName $VMName -AadClientID $AADClientID -AadClientSecret $AADClientSecret -DiskEncryptionKeyVaultUrl $DiskEncryptionKeyVaultUrl -DiskEncryptionKeyVaultId $KeyVaultResourceId -KeyEncryptionKeyUrl $KeyEncryptionKeyUrl -KeyEncryptionKeyVaultId $KeyVaultResourceId -VolumeType $VolumeType
 ```
 
-This example uses Azure AD client ID and client secret to enable encryption on a VM, and wraps the disk encryption key using a key encryption key.
+This example uses Microsoft Entra client ID and client secret to enable encryption on a VM, and wraps the disk encryption key using a key encryption key.
 
-### Example 6: Enable encryption using Azure AD client ID, client cert thumbprint, and wrap disk encryptionkey by using key encryption key
+<a name='example-6-enable-encryption-using-azure-ad-client-id-client-cert-thumbprint-and-wrap-disk-encryptionkey-by-using-key-encryption-key'></a>
+
```
Comment (BethanyZhou) [inline src/Compute/Compute/help/Set-AzVMDiskEncryptionExtension.md:206]:
```suggestion
```

File: src/Compute/Compute/help/Get-AzVMADDomainExtension.md (hunk 1 above)
Comment (CelesteDG) [inline src/Compute/Compute/help/Get-AzVMADDomainExtension.md:22]:
```suggestion
The **Get-AzVMADDomainExtension** cmdlet gets information about the specified Microsoft Entra domain extension.
```

File: src/Compute/Compute/help/Set-AzVMADDomainExtension.md (hunk 2 above)
Comment (CelesteDG) [inline src/Compute/Compute/help/Set-AzVMADDomainExtension.md:24]:
```suggestion
The **Set-AzVMADDomainExtension** cmdlet adds a Microsoft Entra domain virtual machine extension to a virtual machine.
```

**Review by CelesteDG — CHANGES_REQUESTED** [review]

**Comment by BethanyZhou** [comment]
```
```

**Comment by BethanyZhou** [comment]
false positive report for example issues. won't suppress them to allow further debug. require force merge.
```
  "Module","Cmdlet","Example","Line","RuleName","ProblemId","Severity","Description","Extent","Remediation"
  "Az.Compute","Set-AzVMDiskEncryptionExtension","1","4","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","3","6","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","5","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","36","Invalid_Cmdlet","5000","1","Set-AzKeyVaultSecret is not a valid command name.","Set-AzKeyVaultSecret -VaultName $VaultName -Name $KeyVaultSecretName -SecretValue $Secret","Check the spell of Set-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","37","Invalid_Cmdlet","5000","1","Set-AzKeyVaultAccessPolicy is not a valid command name.","Set-AzKeyVaultAccessPolicy -VaultName $VaultName -ResourceGroupName $RGName -EnabledForDeployment","Check the spell of Set-AzKeyVaultAccessPolicy."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","40","Invalid_Cmdlet","5000","1","Get-AzKeyVaultSecret is not a valid command name.","Get-AzKeyVaultSecret -VaultName $VaultName -Name $KeyVaultSecretName","Check the spell of Get-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","41","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","5","8","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","5","14","Invalid_Cmdlet","5000","1","Add-AzKeyVaultKey is not a valid command name.","Add-AzKeyVaultKey -VaultName $VaultName -Name $KEKName -Destination 'Software'","Check the spell of Add-AzKeyVaultKey."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","5","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","9","Invalid_Cmdlet","5000","1","Add-AzKeyVaultKey is not a valid command name.","Add-AzKeyVaultKey -VaultName $VaultName -Name $KEKName -Destination 'Software'","Check the spell of Add-AzKeyVaultKey."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","38","Invalid_Cmdlet","5000","1","Set-AzKeyVaultSecret is not a valid command name.","Set-AzKeyVaultSecret -VaultName $VaultName-Name $KeyVaultSecretName -SecretValue $Secret","Check the spell of Set-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","39","Invalid_Cmdlet","5000","1","Set-AzKeyVaultAccessPolicy is not a valid command name.","Set-AzKeyVaultAccessPolicy -VaultName $VaultName -ResourceGroupName $RGName -EnabledForDeployment","Check the spell of Set-AzKeyVaultAccessPolicy."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","42","Invalid_Cmdlet","5000","1","Get-AzKeyVaultSecret is not a valid command name.","Get-AzKeyVaultSecret -VaultName $VaultName -Name $KeyVaultSecretName","Check the spell of Get-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","43","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  
```

**Review by BethanyZhou — APPROVED** [review]
LGTM. Analyzer is reporting false positive for example issues. Analyzer can't recognize cmdlet from KeyVault module. Won't suppress them to allow further debug. needs to force merge.
could you help force-merge the PR? @isra-fel.

```
"Module","Cmdlet","Example","Line","RuleName","ProblemId","Severity","Description","Extent","Remediation"
  "Az.Compute","Set-AzVMDiskEncryptionExtension","1","4","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","3","6","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","5","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","36","Invalid_Cmdlet","5000","1","Set-AzKeyVaultSecret is not a valid command name.","Set-AzKeyVaultSecret -VaultName $VaultName -Name $KeyVaultSecretName -SecretValue $Secret","Check the spell of Set-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","37","Invalid_Cmdlet","5000","1","Set-AzKeyVaultAccessPolicy is not a valid command name.","Set-AzKeyVaultAccessPolicy -VaultName $VaultName -ResourceGroupName $RGName -EnabledForDeployment","Check the spell of Set-AzKeyVaultAccessPolicy."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","40","Invalid_Cmdlet","5000","1","Get-AzKeyVaultSecret is not a valid command name.","Get-AzKeyVaultSecret -VaultName $VaultName -Name $KeyVaultSecretName","Check the spell of Get-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","4","41","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","5","8","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","5","14","Invalid_Cmdlet","5000","1","Add-AzKeyVaultKey is not a valid command name.","Add-AzKeyVaultKey -VaultName $VaultName -Name $KEKName -Destination 'Software'","Check the spell of Add-AzKeyVaultKey."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","5","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","9","Invalid_Cmdlet","5000","1","Add-AzKeyVaultKey is not a valid command name.","Add-AzKeyVaultKey -VaultName $VaultName -Name $KEKName -Destination 'Software'","Check the spell of Add-AzKeyVaultKey."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","38","Invalid_Cmdlet","5000","1","Set-AzKeyVaultSecret is not a valid command name.","Set-AzKeyVaultSecret -VaultName $VaultName-Name $KeyVaultSecretName -SecretValue $Secret","Check the spell of Set-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","39","Invalid_Cmdlet","5000","1","Set-AzKeyVaultAccessPolicy is not a valid command name.","Set-AzKeyVaultAccessPolicy -VaultName $VaultName -ResourceGroupName $RGName -EnabledForDeployment","Check the spell of Set-AzKeyVaultAccessPolicy."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","42","Invalid_Cmdlet","5000","1","Get-AzKeyVaultSecret is not a valid command name.","Get-AzKeyVaultSecret -VaultName $VaultName -Name $KeyVaultSecretName","Check the spell of Get-AzKeyVaultSecret."
  "Az.Compute","Set-AzVMDiskEncryptionExtension","6","43","Invalid_Cmdlet","5000","1","Get-AzKeyVault is not a valid command name.","Get-AzKeyVault -VaultName $VaultName -ResourceGroupName $RGName","Check the spell of Get-AzKeyVault."

```

**Pull request merged**